R/data.R

```diff
@@ -203,14 +177,17 @@
 #' census records that are screened to protect confidentiality. These extracts
 #' are referred to as public use microdata sample (PUMS) files.*
 #'
-#' *"For the 2010 Census, each state, the District of Columbia, Puerto Rico, and
-#' some Island Area participants delineated PUMAs for use in presenting PUMS
-#' data based on a 5 percent sample of decennial census or American Community
-#' Survey data. These areas are required to contain at least 100,000 people.
-#' This is different from Census 2000 when two types of PUMAs were defined: a 5
-#' percent PUMA as for 2010 and an additional super-PUMA designed to provide a 1
-#' percent sample. The PUMAs are identified by a five-digit census code unique
-#' within state."*
+#' *"PUMAs are statistical geographic areas defined for the tabulation and
+#' dissemination of American Community Survey (ACS) and Puerto Rico Community
+#' Survey Public Use Microdata Sample (PUMS) data as well as ACS period
+#' estimates, and decennial census data. Nesting within states or equivalent
+#' entities, PUMAs cover the entirety of the United States, Puerto Rico, Guam,
+#' and the U.S. Virgin Islands. PUMA delineations are subject to population
+#' threshold criteria, “building block5” geography criteria, and geographic
```

Comment:
Looks like you left a footnote reference (5) from the Census source in this line

Reply:
Oops, thanks, thought I'd gotten all of those but missed that one.